containers/concent-api/Dockerfile.j2

```diff
@@ -4,15 +4,16 @@ COPY testing.pref                   /etc/apt/preferences.d/
 COPY install-dependencies.sh        /tmp/
 RUN  /tmp/install-dependencies.sh
 
-COPY requirements.lock              /srv/http/concent_api/
-COPY middleman_protocol             /srv/http/middleman_protocol/
-COPY install-concent-api.sh         /tmp/
+COPY cacheable-requirements.lock          /srv/http/concent_api/
+COPY install-concent-api.sh               /tmp/
 RUN  /tmp/install-concent-api.sh
-ADD concent-api.tar                 /srv/http/
-COPY concent-api-manage.sh          /usr/local/bin/
-COPY run-celery.sh                  /usr/local/bin/
-COPY run-gunicorn.sh                /usr/local/bin/
-
-COPY local_settings.py              /srv/http/concent_api/concent_api/settings/
+COPY middleman_protocol                   /srv/http/middleman_protocol/
+COPY install-middleman-protocol.sh        /tmp/
+RUN  /tmp/install-middleman-protocol.sh
```

Comment:
I'd put `middleman_protocol/` above Concent. It's going to change much less. And Concent depends on it.

Reply:
You have this:

``` dockerfile
COPY cacheable-requirements.lock          /srv/http/concent_api/
COPY install-concent-api.sh               /tmp/
RUN  /tmp/install-concent-api.sh
COPY middleman_protocol                   /srv/http/middleman_protocol/
COPY install-middleman-protocol.sh        /tmp/
RUN  /tmp/install-middleman-protocol.sh
```

I'm suggesting this:
``` dockerfile
COPY cacheable-requirements.lock          /srv/http/concent_api/
COPY middleman_protocol                   /srv/http/middleman_protocol/
COPY install-middleman-protocol.sh        /tmp/
RUN  /tmp/install-middleman-protocol.sh
COPY install-concent-api.sh               /tmp/
RUN  /tmp/install-concent-api.sh
```

Reply:
After discussing this I see what you mean. `install-concent-api.sh` only installs the Python dependencies. In that case we should just change its name to `install-concent-api-python-dependencies.sh` to make it clear.